queries/unison/highlights.scm

```diff
@@ -0,0 +1,75 @@
+;; Primitives
+(comment) @comment
+(nat) @constant.numeric
+(unit) @constant.builtin
+(literal_char) @constant.character
+(literal_text) @string
+(literal_boolean) @constant.builtin.boolean
+
+;; Keywords
+[
+  (kw_forall)
+  (unique_kw)
+  (type_kw)
+  (kw_equals)
+  (do)
+] @keyword
+
+(kw_let) @keyword.function
+(structural_kw) @keyword.storage.type
+(unique) @keyword.storage.modifier
+
+[ 
+  (type_constructor)
+] @constructor
+
+(doc_block) @comment
+
+
+[
+  (operator)
+  (pipe) 
+  (arrow_symbol) 
+  (">")
+  (or)
+] @operator
+
+[
+  "if"
+  "else"
+  "then"
+  (match)
+  (with)
+  (cases)
+] @keyword.control.conditional
+
+(blank_pattern) @variable.builtin
+
+((pattern) @variable.other.member (constructor_or_variable_pattern) @type)
+
+(use_clause) @keyword.control.import
+
+;; Types
+(record_field name: (wordy_id) @variable.other.member type: (wordy_id) @type)
```

Comment:
I will take this as an example of following `CONTRIBUTING.md`

Only `@variable`, not the suffix .other.member